### DISCUSSION TITLE
Add export functionality in the wallets directly or make it easier for users to go the the explorer page for a specific address.

### DISCUSSION THREAD

**Discussion by MaudSimon** [alephium/alephium-frontend#465] [Feature requests]
From Discord:

> Being new to the wallet/project it's quite odd to me that there is no export in the wallet.  I'm glad you chose to have it in the Explorer at least, but then there is no link ot the Explorer in the wallet.
> 
> That I think you can/should fix - Please put a link to the block Explorer somewhere easy to find in the wallet! 😊
> 
> Otherwise, really cool project looking forward to continue mining.
> Thanks to the devs, congrats and luck and success to you all.

**Comment by nop33** [comment]
The desktop wallet provides the export feature. It also provides a link to the explorer.

![image](https://github.com/alephium/alephium-frontend/assets/1579899/090350b9-8d2b-49c0-94a1-8643fa5c2b5d)


![image](https://github.com/alephium/alephium-frontend/assets/1579899/16ff87bd-168d-4c02-8a80-f1061dd15bd9)


I agree that in the mobile wallet the address card should have a link to the explorer.
